azerothcore/azerothcore-wotlk
# fix(DB/Spells): Arcane Missiles: They should not cause aggro when launched, but when hit with the first beam.

## Changes Proposed:
-  Set flag NO_INITIAL_AGGRO for arcane missiles.

## SOURCE:



## Tests Performed:

- Compile and works in windows 10 and ubuntu 20.


## How to Test the Changes:

- log in with a mage and try to launch each and every one of the arcane missile ranges on an enemy and notice that now the power only generates aggro when it hits the enemy and not just the casting starts.

## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
